### PR TITLE
Add base generic inference and enable token classification

### PR DESCRIPTION
**Pull request by osanseviero**
FYI @lewtun @nateraw 

Repo template: https://huggingface.co/templates/token-classification
Repo example: https://huggingface.co/osanseviero/en_core_web_sm

### CODE REVIEW

File: api-inference-community/docker_images/generic/requirements.txt [hunk 1]
```diff
@@ -1,2 +1,8 @@
 starlette==0.14.2
-api-inference-community==0.0.8
+api-inference-community==0.0.13
+torch>=1.7.0
+scikit-learn>=0.23.2
+scipy>=1.5.4
+numba>=0.48
```
Comment (lewtun) [inline api-inference-community/docker_images/generic/requirements.txt:6]: out of curiosity, why do we need `numba` in the base requirements?

**Review by lewtun — APPROVED** [review]
this looks really great - i'm excited to see what people do with the generic interface :) 

not sure if this is a problem or not, but the repo example doesn't seem to be compatible with the widget (some files are missing?)

Screen Shot 2021-08-02 at 09 18 03

**Comment by osanseviero** [comment]
> not sure if this is a problem or not, but the repo example doesn't seem to be compatible with the widget (some files are missing?)

This is not deployed yet, so it's currently trying to use `transformers` instead, hence the error.

File: api-inference-community/docker_images/generic/requirements.txt (hunk 1 above)
Comment (osanseviero) [reply api-inference-community/docker_images/generic/requirements.txt]: Good question, I removed it. I still need to decide which will be the pre-installed requirements and their respective versions, so this is mostly a temporal placeholder.

**Comment by KoichiYasuoka** [comment]
Thank you @osaseviero to implement token-classification. I could use it with BertTokenizer, but I could not use with BertJapaneseTokenizer; neither in https://huggingface.co/KoichiYasuoka/bert-base-japanese-upos nor in https://huggingface.co/KoichiYasuoka/bert-large-japanese-upos now. In the JSON-cache of `bert-large-japanese-upos` I found that `start` and `end` gone `null` but I'm vague how to fix...

**Comment by osanseviero** [comment]
Hi @KoichiYasuoka!

This issue seems unrelated to this PR and might go better in the `transformers` repository as a new issue. 

Anyways, I took a quick look at your repo, and the error is the following
`Asking to truncate to max_length but no maximum length is provided and the model has no predefined maximum length. Default to no truncation.`, but even specifying this there seems to be some issues. Please open an issue in https://github.com/huggingface/transformers/issues and we can go from there.

**Comment by KoichiYasuoka** [comment]
Thank you @osanseviero it works today!

```
>>> from huggingface_hub.inference_api import InferenceApi
>>> api=InferenceApi("KoichiYasuoka/bert-base-japanese-upos")
>>> api(inputs="国境の長いトンネルを抜けると雪国であった。")
[{'entity_group': 'NOUN', 'score': 0.9992469549179077, 'word': '国 境', 'start': None, 'end': None}, {'entity_group': 'ADP', 'score': 0.9997754693031311, 'word': 'の', 'start': None, 'end': None}, {'entity_group': 'ADJ', 'score': 0.9959967136383057, 'word': '長 い', 'start': None, 'end': None}, {'entity_group': 'NOUN', 'score': 0.999525785446167, 'word': 'ト ン ネ ル', 'start': None, 'end': None}, {'entity_group': 'ADP', 'score': 0.9997870326042175, 'word': 'を', 'start': None, 'end': None}, {'entity_group': 'VERB', 'score': 0.9990143179893494, 'word': '抜 け る', 'start': None, 'end': None}, {'entity_group': 'CCONJ', 'score': 0.9952036738395691, 'word': 'と', 'start': None, 'end': None}, {'entity_group': 'NOUN', 'score': 0.990840494632721, 'word': '雪 国', 'start': None, 'end': None}, {'entity_group': 'AUX', 'score': 0.9994673132896423, 'word': 'で', 'start': None, 'end': None}, {'entity_group': 'AUX', 'score': 0.9990374445915222, 'word': 'あ っ', 'start': None, 'end': None}, {'entity_group': 'AUX', 'score': 0.9997127056121826, 'word': 'た', 'start': None, 'end': None}, {'entity_group': 'PUNCT', 'score': 0.9998103976249695, 'word': '。', 'start': None, 'end': None}]
```

Thank you again (for whom might fix this).

**Pull request merged**